spec/sql_footprint_spec.rb

```diff
@@ -28,14 +29,14 @@
       )
     end
 
+    it 'formats selects by 1 count' do
```

Comment:
This test isn't checking the formatting so this might be a better name:
`it 'appends 1 statement'`